overlays/tls-overlay.yaml

```diff
@@ -4,10 +4,26 @@ applications:
     channel: edge
     scale: 1
     options:
-      ca-common-name: ca.demo.local
+      ca-common-name: <your-domain-name>
```

Comment:
- This file is not a template, and should be deployable out of the box. I hope the invalid `<your-domain-name>` string would at least result in a blocked status.
- Iirc, there are itests that rely on having `ca.demo.local` there.